jwst/datamodels/schemas/multislit.schema.yaml

```diff
@@ -88,4 +88,12 @@ allOf:
             type: number
             fits_keyword: STLARITY
             fits_hdu: SCI
+          pl_point:
+            fits_hdu: PATHLOSS_POINT
+            default: 1.0
+            datatype: float32
+          pl_uni:
```

Comment:
A minor nit, but for the purpose of code readability it would be more obvious to expand this attribute name to "pl_uniform" (it's only a few more characters).